src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -9769,14 +9783,11 @@ private static bool IsNullabilityMismatch(TypeSymbol type1, TypeSymbol type2)
             return result;
         }
 
-        public override BoundNode? VisitInterpolatedString(BoundInterpolatedString node)
-        {
-            // https://github.com/dotnet/roslyn/issues/54583
-            // Better handle the constructor propogation
-            var result = base.VisitInterpolatedString(node);
-            SetResultType(node, TypeWithState.Create(node.Type, NullableFlowState.NotNull));
-            return result;
-        }
+        // https://github.com/dotnet/roslyn/issues/54583
+        // Better handle the constructor propogation
+        //public override BoundNode? VisitInterpolatedString(BoundInterpolatedString node)
```

Comment:
Nit: can we get rid of the commented code? either enter the function and return node, or remove.

Reply:
I intentionally left this in to leave a place for the issue comment.